### ISSUE TITLE
Ints vs strings for person ids

### ISSUE THREAD

**Issue by f-hafner** [odissei-lifecourse/life-sequencing-dutch#80]
In the data used for the evaluation (for instance in the pickle files), we're storing the person identifiers as integers. we should check if this is safe because some ids may have leading 0s.

**Comment by tanzir5** [comment]
Some ids indeed has leading zeros in the raw data. But I have checked using [this code](https://github.com/odissei-lifecourse/life-sequencing-dutch/blob/main/pop2vec/evaluation/check_rinpersoon_int.py) that the leading zeros do not create a collision when converted to integers. So there is no such scenario where two persons have ids like "00119" and "119". 

1. Ran the code on the RA machine.
2. Ran on all the inpatab files from 2011 to 2022 using pyreadstat.read_sav(). The df assumes the rinpersoon column has dtype object.
